### ISSUE TITLE
TypeError: init_inference() got an unexpected keyword argument 'base_dir' in bloom-ds-inference.py

### ISSUE THREAD

**Issue by Oogy** [huggingface/transformers-bloom-inference#5]
When running
```sh
deepspeed --num_gpus 8 bloom-inference-scripts/bloom-ds-inference.py --name microsoft/bloom-deepspeed-inference-fp16 --benchmark
```
I receive the following error:
```sh0.7.3
Traceback (most recent call last):
  File "bloom-ds-inference.py", line 187, in <module>
    model = deepspeed.init_inference(model,
TypeError: init_inference() got an unexpected keyword argument 'base_dir'
```
After removing the `base_dir` arg from [this line](https://github.com/huggingface/transformers-bloom-inference/blob/main/bloom-inference-scripts/bloom-ds-inference.py#L187) and re-running I get this error:
```sh
NotImplementedError: Cannot copy out of meta tensor; no data!
Traceback (most recent call last):
  File "bloom-inference-scripts/bloom-ds-inference.py", line 187, in <module>
    model = deepspeed.init_inference(model,
  File "/usr/local/lib/python3.8/dist-packages/deepspeed/__init__.py", line 292, in init_inference
    engine = InferenceEngine(model,
  File "/usr/local/lib/python3.8/dist-packages/deepspeed/inference/engine.py", line 151, in __init__
    self.module.to(device)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 927, in to
    return self._apply(convert)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 579, in _apply
    module._apply(fn)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 579, in _apply
    module._apply(fn)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 602, in _apply
    param_applied = fn(param)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 925, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
NotImplementedError: Cannot copy out of meta tensor; no data!
```
Running `deepspeed==0.7.2`. Please advise.

**Edit:** Apologies, pasted the wrong error in second snippet. Updated.

**Comment by mayank31398** [comment]
@Oogy you need to use deepspeed>=0.7.3
Alternatively, you can build from master branch

**Comment by Oogy** [comment]
@mayank31398 Latest available when the issue was opened was `0.7.2`, `0.7.3` only released yesterday. However after upgrading it is now working. Thank you!
